### ISSUE TITLE
Travis: network timeout after TLS handshake

### ISSUE THREAD

**Issue by bassosimone** [measurement-kit/mkcollector#23]
Not a super big deal. It seems it's always timing it during some tests. Running tests locally actually work and running them on other cloud servers also work. Transient issue?

**Comment by bassosimone** [comment]
Here's a relevant excerpt from [a failed build logs](https://travis-ci.org/measurement-kit/mkcollector/jobs/538148077):

```
=== BEGIN CLOSE LOGS ===
  Trying 37.218.241.178...
TCP_NODELAY set
Connected to b.collector.ooni.io (37.218.241.178) port 443 (#0)
ALPN, offering h2
ALPN, offering http/1.1
successfully set certificate verify locations:
  CAfile: .mkbuild/download/ca-bundle.pem
  CApath: /etc/ssl/certs
>tls_data: 5
TLSv1.3 (OUT), TLS handshake, Client hello (1):
>tls_data: 512
<tls_data: 5
TLSv1.3 (IN), TLS handshake, Server hello (2):
<tls_data: 111
NPN, negotiated HTTP1.1
<tls_data: 5
TLSv1.2 (IN), TLS handshake, Certificate (11):
<tls_data: 2563
<tls_data: 5
TLSv1.2 (IN), TLS handshake, Server key exchange (12):
<tls_data: 333
<tls_data: 5
TLSv1.2 (IN), TLS handshake, Server finished (14):
<tls_data: 4
>tls_data: 5
TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
>tls_data: 70
>tls_data: 5
TLSv1.2 (OUT), TLS change cipher, Change cipher spec (1):
>tls_data: 1
>tls_data: 5
TLSv1.2 (OUT), TLS handshake, Next protocol (67):
>tls_data: 36
>tls_data: 5
TLSv1.2 (OUT), TLS handshake, Finished (20):
>tls_data: 16
Operation timed out after 14000 milliseconds with 0 out of 0 bytes received
Closing connection 0
=== END CLOSE LOGS ===
```

There is a timeout after 14 seconds of inactivity after the handshake.

The issue seems persistent. That is, restarting the builds always causes it to happen.

**Comment by bassosimone** [comment]
This should have been fixed in #25 where I also applied a `docker_tc_disabled: true` change to `MKBuild.yaml`, so we should not have insane latency anymore. Will reopen on relapse.
